lib/src/http_unmodifiable_map.dart

```diff
@@ -53,6 +53,7 @@ class HttpUnmodifiableMap<V> extends UnmodifiableMapView<String, V> {
 /// A const implementation of an empty [HttpUnmodifiableMap].
 class _EmptyHttpUnmodifiableMap<V> extends MapView<String, V>
     implements HttpUnmodifiableMap<V> {
-  bool get _ignoreKeyCase => true;
   const _EmptyHttpUnmodifiableMap() : super(const {});
+
+  bool get _ignoreKeyCase => true;
```

Comment:
I can change it back if you desire. I like the rule because it keeps things consistent.